doc/manual/manual.tex

```diff
@@ -10872,6 +10872,43 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
 
 
+\subsubsection{Brittle thrust wedges benchmark}
+\label{sec:benchmark-brittle-thrust-wedge}
+
+\textit{This section was contributed by Sibiao Liu, Stephanie Sparks, John Naliboff, and Cedric Thieulot.}
+
+Buiter et al. (2016, \cite{buiter16}) organized new comparison experiments with analogue and numerical models to investigate brittle thrust wedge behavior. The benchmark here aims to verify that the wedge models using  \aspect{} follows other numerical results and the analytical wedge theory showing in this paper. In particular, input files (\url{benchmarks/ buiter_et_al_2016_jsg}) are provided for reproducing the numerical simulations of stable wedge experiment 1 and unstable wedge experiment 2 with same model setups.
+
+A number of model sets of prescribed material behaviorl are required to simulate the brittle thrust formation. For example, although the material in the numerical model has a visco-plastic rheology, it performs plastic yielding at the beginning of shortening due to the non-viscos sand. The plastic strain-weakening behavior, with the internal angle of friction reducing between total finite strain invariant values of 0.5 and 1.0, is prescribed to mimic the softening from peak to dynamic stable strength correlates with sand dilation. 
+
+In sandbox-type models an important role is played by the boundaries and the frictional sliding of sand against these boundaries. For the top boundary condition, zero traction ("open") and a sticky air layer is used to approximate a free surface. Additional testing revealed that using a true free surface leads to significant mesh distortion and associated numerical instabilities. We apply a rigid block that approximates a mobile wall with a constant velocity of 2.5 cm/hour on the right-hand side boundary to drive the deformation in sand layers. 
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/brittle_thrust_wedges/velocity_bc_part.prm}
+
+A good and accurate solver convergence is always challenging to achieve in numerical thrust wedge models with a high spatial resolution (ca. 1 mm node spacing) and a large viscosity contrast. Here, we suggest that several parameters should be considered carefully. Firstly, the nonlinear and linear solver tolerances should be sufficiently strict to avoid numerical instabilities. Secondly, we use the discontinuous Galerkin method (set Use discontinuous composition discretization = true) to ensure that the discontinuous composition bound preserving limiter produces sharp interfaces between compositional layers. Lastly, using the harmonic averaging scheme for material and viscosity is required to achieve reasonable convergence behavior.
```

Comment:
```suggestion
A good and accurate solver convergence is always challenging to achieve in numerical thrust wedge models with a high spatial resolution (ca.~1 mm node spacing) and a large viscosity contrast. Here, we suggest that several parameters should be considered carefully. Firstly, the nonlinear and linear solver tolerances should be sufficiently strict to avoid numerical instabilities. Secondly, we use the discontinuous Galerkin method (\texttt{set Use discontinuous composition discretization = true}) to ensure that the discontinuous composition bound-preserving limiter produces sharp interfaces between compositional layers. Lastly, using the harmonic averaging scheme for material and viscosity is required to achieve reasonable convergence behavior.
```

Reply:
"A good and accurate solver convergence is always challenging to achieve in numerical thrust wedge models with a high spatial resolution (ca.~1 mm node spacing) and a large viscosity contrast. Here, we suggest that several parameters should be considered carefully. Firstly, the nonlinear and linear solver tolerances should be sufficiently strict to avoid numerical instabilities. Secondly, we use the discontinuous Galerkin method (\texttt{set Use discontinuous composition discretization = true}) to ensure that the discontinuous composition bound-preserving limiter produces sharp interfaces between compositional layers. Lastly, using the harmonic averaging scheme for material and viscosity is required to achieve reasonable convergence behavior."->

"**Accurate** solver convergence is always challenging to achieve in numerical thrust wedge models with a high spatial resolution (ca.~1 mm node spacing) and a large viscosity contrast. Here, we suggest that several parameters should be considered carefully. **First**, the nonlinear and linear solver tolerances should be sufficiently strict to avoid numerical instabilities. **Second**, we use the discontinuous Galerkin method (\texttt{set Use discontinuous composition discretization = true}) to ensure that the discontinuous composition bound-preserving limiter produces sharp interfaces between compositional layers. Lastly, **we use** the harmonic averaging scheme for material and **viscosity to** achieve reasonable convergence behavior."